js/app.js

```diff
@@ -6,7 +6,7 @@ $(document).ready(function () {
   }
 
   // Unveil images 50px before they appear
```

Comment:
Does this code comment need to be updated? Does jquery.lazy still use 50px as the padding for lazy loading?